docs/release_process.rst

```diff
@@ -0,0 +1,195 @@
+    # PR description isn't important)
+    # Merge this PR immediately to kick off the build
+    # Create PR on the ``release`` branch named ``rc/0.10.0`` using the
+    # console output from git-release-notes v0.9.0..master util/release_notes.ejs
+    # as the PR description
+    # Once the ``release`` branch deploys on https://lore-rc.herokuapp.com ,
+    # Instruct developers to view the ``release`` branch PR description and
+    # validate their commits on https://lore-rc.herokuapp.com
+    # Wait for all checkboxes to be checked
+    # Check with Dev-Ops buddy whether migrations need to be run for this release
+    git flow release finish 0.10.0
+    git push --tags
+    # Send release notification to LORE mailing list - odl-lore-announce@mit.edu
+
+Detailed Instructions
+----------------------
+
+Create, commit, and push release candidate branch
+=================================================
+
+.. code-block:: bash
+
+    git fetch
+    git checkout master
+    git pull
+    git flow release start 0.10.0
+
+Update the ``VERSION`` number in ``lore/settings.py`` following Semantic
+numbering practice.
+
+.. code-block:: bash
+
+    vi lore/settings.py # change version number
+
+Generate the `release notes`_ in reStructuredText format:
+
+.. code-block:: bash
+
+  git-release-notes v0.9.0..master util/release_notes_rst.ejs
+
+Update ``release.rst`` with this version's release notes (the console output
+generated by running ``release_notes_rst.ejs``).  Edit the new
+release notes to correct the release number, make each statement in the
+past tense, capitalize the first word, and end sentences with a period.
+
+.. code-block:: bash
+
+    vi RELEASE.rst
+
+Add, commit, and push the two files you have updated to publish
+the release.
+
+.. code-block:: bash
+
+    git add RELEASE.rst lore/settings.py
+    git commit -m "Release 0.10.0"
+    git flow release publish 0.10.0
+
+The "release publish" (push) will trigger a build on Travis CI.
+https://travis-ci.org/mitodl/lore/builds  For each commit throughout
+the release, it's good practice to check that the build succeeded
+before proceeding.
+
+Create Pull Requests
+====================
+
+Developers will start their verification tests as soon as
+they learn that you've created the PR with the checkboxes.  Since
+the deployment to ``lore-rc`` can take a while, merge the
+``release-candidate`` branch PR as soon as you create it to move
+its deployment off the critical path.
+
+Make three PRs based on the release branch you just published:
+
+  - One to the ``release-candidate`` branch named ``rc/0.10.0``
+  - Merge this PR immediately to kick off the build.
+  - One to the ``release`` branch named ``rc/0.10.0`` containing a checklist of commits
+  - one to the ``master`` branch named ``Release 0.10.0``
+
+Run ``git-release-notes`` with the other template to create the
+Pull Request descriptions for the ``release`` branch.
+
```

Comment:
I still think you want to create the PR descriptions before opening the PR, i.e. put lines 110-11 before 103.